### PR TITLE
Update 08-putting-it-all-together.md

### PR DESCRIPTION
**Pull request by emilyyaklich**
Update the deprecated links to Matplotlib plot examples.

### CODE REVIEW

File: episodes/08-putting-it-all-together.md [hunk 1]
```diff
@@ -146,7 +146,7 @@ for the rest.
 
 We will cover a few basic commands for creating and formatting plots with matplotlib in this lesson.
 A great resource for help creating and styling your figures is the matplotlib gallery
-([http://matplotlib.org/gallery.html](https://matplotlib.org/gallery.html)), which includes plots in many different
+([[http://matplotlib.org/gallery.html](https://matplotlib.org/stable/gallery/index.html)](https://matplotlib.org/stable/gallery/index.html)), which includes plots in many different
```
Comment (btovar) [inline episodes/08-putting-it-all-together.md:149]: I wonder if this one should be:
```md
the [matplot gallery](https://matplotlib.org/stable/gallery/index.html), which includes ...
```

**Review by btovar — CHANGES_REQUESTED** [review]

File: episodes/08-putting-it-all-together.md (hunk 1 above)
Comment (quist00) [reply episodes/08-putting-it-all-together.md:149]: I think these are important link changes that are in limbo.  Will try to resolve this pull request with minor edits.

Comment (quist00) [inline episodes/08-putting-it-all-together.md:149]:
```suggestion
([matplot gallery](https://matplotlib.org/stable/gallery/)), which includes plots in many different
```

File: episodes/08-putting-it-all-together.md [hunk 2]
```diff
@@ -361,7 +361,7 @@ fig.savefig("my_plot_name.pdf", dpi=300)
 ## Make other types of plots:
 
 Matplotlib can make many other types of plots in much the same way that it makes two-dimensional line plots. Look through the examples in
-[http://matplotlib.org/users/screenshots.html](https://matplotlib.org/users/screenshots.html) and try a few of them (click on the
+[https://matplotlib.org/stable/plot_types/index.html](https://matplotlib.org/stable/plot_types/index.html) and try a few of them (click on the
```
Comment (quist00) [inline episodes/08-putting-it-all-together.md:364]:
```suggestion
[https://matplotlib.org/stable/plot_types/](https://matplotlib.org/stable/plot_types/) and try a few of them (click on the
```

**Review by quist00 — APPROVED** [review]

**Pull request merged**